src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  shape: Shape,
+  color: string,
+  area: number,
+  getArea(): void;
 }
 
 export class Triangle {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const bigest: number = Math.max(this.a, this.b, this.c);
+
+    const sum: number = this.a + this.b + this.c;
+
+    if (this.a <= 0
+      || this.b <= 0
+      || this.c <= 0
+      || (sum - bigest) <= bigest) {
+      throw new Error('The params should be bigger than zero.');
+    }
```

Comment:
it would be better to split into 2 checks (first for sides bigger than zero and the second: 2 sides shouldn't be bigger than third)